astroquery/gaia/core.py

```diff
@@ -19,10 +19,25 @@
 from astroquery.utils import commons
 from astropy import units
 from astropy.units import Quantity
+from astropy import config as _config
+import six
+from astroquery.utils.tap import taputils
 
-from . import conf
 
-__all__ = ['Gaia', 'GaiaClass']
+class Conf(_config.ConfigNamespace):
+    """
+    Configuration parameters for `astroquery.gaia`.
+    """
+
+    MAIN_GAIA_TABLE = _config.ConfigItem("gaiadr2.gaia_source",
+                                         "GAIA source data table")
+    MAIN_GAIA_TABLE_RA = _config.ConfigItem("ra",
+                                            "Name of RA parameter in table")
+    MAIN_GAIA_TABLE_DEC = _config.ConfigItem("dec",
+                                             "Name of Dec parameter in table")
+
+
+conf = Conf()
```

Comment:
Why did you move the configuration here instead of leaving it in `__init__`?

Reply:
I will move conf to __init__.py.
Thanks.